src/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star-active.svg);
+  background-position: contain;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n+5) {
+  background-image: url(../src/images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(2),
+.stars--1 .stars__star:nth-child(3),
+.stars--1 .stars__star:nth-child(4),
+.stars--1 .stars__star:nth-child(5) {
+  background-image: url(../src/images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(3),
+.stars--2 .stars__star:nth-child(4),
+.stars--2 .stars__star:nth-child(5) {
+  background-image: url(../src/images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(4),
+.stars--3 .stars__star:nth-child(5) {
+  background-image: url(../src/images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(5) {
+  background-image: url(..//src/images/star.svg);
+}
```

Comment:
2'nd point in checklist

Reply:
this isn't fixed, it will work like this
```
.stars--1 :first-child,
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: url(images/star-active.svg);
}